src/main.js

```diff
@@ -41,10 +41,17 @@ app.on('ready', () => {
     windows.about.snippetsManager = snippetsManager
     windows.preferences.snippetsManager = snippetsManager
     windows.preferences.preferencesManager = preferencesManager
+    windows.popup.snippetsManager = snippetsManager
 
-    windows.preferences.on('focus', () => { snippetsManager.shouldMatch = false })
+    windows.preferences.on('focus', () => {
+        snippetsManager.shouldMatch = false
+        windows.popup.hide()
```

Comment:
I close the popup on preferences window focus, so that a user cannot remove a snippet with the popup open. I guess it doesn't matter to a user to have that happen except a situation where they'd like to move the preferences window. What do you think?